examples/secp256k1-transfer/package.json

```diff
@@ -15,8 +15,8 @@
     "@ckb-lumos/lumos": "0.20.0-alpha.1",
     "@types/react": "^18.0.25",
     "@types/react-dom": "^18.0.9",
-    "react": "18.2.0",
-    "react-dom": "18.2.0"
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
```

Comment:
we can keep this in this PR, so that we don't need to modify the lock files in this PR